src/saveload/afterload.cpp

```diff
@@ -3273,6 +3273,13 @@ bool AfterLoadGame()
 		}
 	}
 
+	if (IsSavegameVersionBeforeOrAt(SLV_CALENDAR_SUB_DATE_FRACT)) {
```

Comment:
Please create a new `SLV_` for this PR; no need to piggyback on another version.